definitions/windows.yaml

```diff
@@ -645,6 +755,20 @@ sources:
   attributes: {query: SELECT * FROM Win32_Service}
 supported_os: [Windows]
 ---
+name: WinAppXRT
+doc: WinAppXRT DLL loaded when the APPX_PROCESS environment variable is set.
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%environ_systemroot%%\system32\WinAppXRT.dll'
```

Comment:
This is OK for now, but it would be even better if you could just do:

```%%environ_path%%\\WinAppXRT.dll```

and have GRR collect environ_path (which it already does) and then figure out all the potential path combinations based on that (which it doesn't do). Filed https://github.com/google/grr/issues/250